catalyst_voices/apps/voices/lib/widgets/buttons/voices_buttons.dart

```diff
@@ -15,6 +15,7 @@ class DrawerToggleButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return VoicesIconButton(
+      key: const Key('drawer_button'),
```

Comment:
Sometimes you call these keys using PascalCase, sometimes with snake_case, can we unify it?

Reply:
Done, switched all to PascalCase, thanks